### PR TITLE
Java: Add container flow to the local taint flow relation.

### PR DESCRIPTION
**Pull request by aschackmull**
We haven't really paid a lot of attention to the local taint relation, and when container flow was improved to be tracked more precisely for global flow, the local version lost the ability to track it. So we add it back as ad-hoc taint steps in the local taint flow relation.

### CODE REVIEW

**Comment by aschackmull** [comment]
This btw. aligns Java local taint flow more closely with the C# version.

**Review by atorralba — COMMENTED** [review]
LGTM, but maybe you'll want to wait for an additional review from someone with more knowledge about the intricacies of the taint tracking library.

**Review by hvitved — APPROVED** [review]

**Pull request merged**
